src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,10 @@
+%centre-flex {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+%resetMarginPadding {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
you'd rather create a file with resetting, could be something like the following
```
li, 
ul, 
h1,
... {
  margin: 0;
}
```